functional/FunctionalTest.java

```diff
@@ -3558,9 +3559,8 @@ public static void main(String[] args) throws Exception {
     } else {
       kmsKeyName = System.getenv("MINIO_JAVA_TEST_KMS_KEY_NAME");
       if (kmsKeyName == null) {
-        kmsKeyName = System.getenv("MINT_KEY_ID");
+        kmsKeyName = System.getenv("MINT_KMS_KEY_NAME");
       }
-      sqsArn = System.getenv("MINIO_JAVA_TEST_SQS_ARN");
```

Comment:
Any reason for removing this?

Reply:
There is no purpose for this @balamurugana - can you explain why this exists?

Reply:
It is needed when running functional tests manually `./gradlew runFunctionalTest -Pendpoint=ENDPOINT -PaccessKey=ACCESSKEY -PsecretKey=SECRETKEY -Pregion=REGION`; otherwise tests depend on `sqsArn` wouldn't run. There is no code without its purpose.

Reply:
Are you expecting users to know all of this run functional tests?

Reply:
Yes. In fact users never run functional tests, but developers do. If a developer is indent to testing `minio-java` himself apart from CI, he has an option to do so. Also note that, `Mint` runs functional tests to test minio server by this way.

Reply:
Yes but this is completely wrong,  one way is env, and another way is command line. It looks clunky way to test things. Why have this option in the first place?